eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/batch/io/deprecated/DeprecatedVariantMongoReader.java

```diff
@@ -107,29 +107,19 @@ protected List<Variant> getVariants(Document deprecatedVariant) {
         Document inactiveEntity = inactiveObjects.iterator().next();
         String contig = inactiveEntity.getString(VariantMongoAggregationReader.CONTIG_FIELD);
         long start = inactiveEntity.getLong(VariantMongoAggregationReader.START_FIELD);
-        String type = inactiveEntity.getString(TYPE_FIELD);
-
-        for (Document submittedVariantOperation : submittedVariantOperations) {
-            Collection<Document> inactiveEntitySubmittedVariant = (Collection<Document>) submittedVariantOperation
-                    .get("inactiveObjects");
-            Document submittedVariant = inactiveEntitySubmittedVariant.iterator().next();
-            long submittedVariantStart = submittedVariant.getLong(START_FIELD);
-            String submittedVariantContig = submittedVariant.getString(CONTIG_FIELD);
-            String reference = submittedVariant.getString("ref");
-            String alternate = submittedVariant.getString("alt");
-
-            if (isSameLocation(contig, start, submittedVariantContig, submittedVariantStart, type)) {
-                // Since we only need evidence that at least one submitted variant agrees
-                // with the deprecated RS in locus, we just return one variant record per RS
-                Variant variantToReturn = new Variant(contig, start,
-                                                      start + Math.max(reference.length(), alternate.length()) - 1,
-                                                      reference, alternate);
-                variantToReturn.setMainId("rs" + deprecatedVariant.getLong("accession"));
-                return Arrays.asList(variantToReturn);
-            }
-        }
-
-        return new ArrayList<>();
+        Collection<Document> inactiveEntitySubmittedVariant = (Collection<Document>) submittedVariantOperations
+                .iterator().next().get("inactiveObjects");
+        Document submittedVariant = inactiveEntitySubmittedVariant.iterator().next();
+        String reference = submittedVariant.getString("ref");
+        String alternate = submittedVariant.getString("alt");
+
+        // Since we only need evidence that at least one submitted variant agrees
+        // with the deprecated RS in locus, we just return one variant record per RS
```

Comment:
We should remove the part about agreeing on locus, and perhaps move the comment to above line 110 as it helps explain why we only call `submittedVariantOperations.iterator().next()` rather than iterating through all operations.

(This is all a bit of a faff since AFAICT we just [output the rsID](https://github.com/EBIvariation/eva-accession/blob/4bbf2066781c756caca3c9ed206789c427f60ba3/eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/batch/io/deprecated/DeprecatedVariantAccessionWriter.java#L69) anyway and the whole aggregation pipeline is to confirm that the rsID was ever present for this taxonomy to begin with... but that's a separate discussion...)

Reply:
done